modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -67,6 +68,36 @@ def query_params(url)
       expect(json['meta']['distances'].length).to eq(10)
     end
 
+    it 'responds to GET #index with zip' do
+      setup_pdx
+      get base_query_path + zip, params: nil, headers: accept_json
+      expect(response).to be_success
+      expect(response.body).to be_a(String)
+      json = JSON.parse(response.body)
+      expect(json['data'].length).to eq(4)
+      expect(json['meta']['distances']).to eq([])
```

Comment:
is this always expected to be empty or is it a stand in until another feature is done?